includes/admin/class-wc-rest-payments-settings-controller.php

```diff
@@ -107,7 +97,7 @@ public function register_routes() {
 					],
 					'digital_wallets_enabled_locations' => [
 						'description'       => __( '1-click checkout locations that should be enabled.', 'woocommerce-payments' ),
-						'type'              => 'object',
+						'type'              => 'array',
 						'validate_callback' => __CLASS__ . '::validate_digital_wallets_enabled_locations',
```

Comment:
Doing this will allow us to remove the `validate_digital_wallets_enabled_locations` method:

```suggestion
						'type'              => 'array',
						'items'             => [
							'type' => 'string',
							'enum' => Digital_Wallets_Locations::toArray(),
						],
						'validate_callback' => 'rest_validate_request_arg',
```

Reply:
Thanks! I was trying all different ways of using `enum` with `Digital_Wallets_Locations` and it kept failing the validation - so I just thought "screw it, I'll just keep the validation callback" 😂 
Done in 7c997e5c